db/migrate/20170209203904_orchestration_stack_enhancements_for_ansible_service_provision.rb

```diff
@@ -0,0 +1,14 @@
+class OrchestrationStackEnhancementsForAnsibleServiceProvision < ActiveRecord::Migration[5.0]
+  def change
+    add_column :orchestration_stacks, :playbook_name, :string
```

Comment:
I think that in this case, duplication of the data is not wanted. If the 3 queries will be an issue, you can do :join instead of :include and you have it in one query.

It make sense to duplicate the data in a case when we e.g. use an Ancestry or a Relationship and we are not able to do a simple join.

@kbrock ^

Also, you got me scared with the binary blob lookup, we do that anywhere?

For the other cols, I think we either do the serialized columns (:jsonb rules there) or CustomAttribute